apps/dotcom/src/utils/useFileSystem.tsx

```diff
@@ -128,7 +133,12 @@ export function getSaveFileCopyAction(
 		kbd: '$s',
 		async onSelect(source) {
 			handleUiEvent('save-project-to-file', { source })
-			const defaultName = saveFileNames.get(editor.store) || `Untitled${TLDRAW_FILE_EXTENSION}`
+			const documentName =
+				editor.getDocumentSettings().name === ''
```

Comment:
q: is `editor.getDocumentSettings().name` always available? should this check for `!editor.getDocumentSettings().name`?

Reply:
It should be.  The signature of `getDocumentSettings` says it always returns `TLDocument` and that one always has a `name` prop.